src/components-examples/material/sidenav/sidenav-responsive/sidenav-responsive-example.ts

```diff
@@ -29,16 +36,27 @@ export class SidenavResponsiveExample implements OnDestroy {
        cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.`,
   );
 
-  private _mobileQueryListener: () => void;
+  mobileQuery: boolean;
```

Comment:
I think that this and `_breakpointSubscription` aren't used anymore and can be removed.